content/chat/rooms/messages.textile

```diff
@@ -224,10 +230,27 @@ const MyComponent = () => {
 };
 ```
 
+```[swift]
+let originalMessage: Message
+let updatedMessage = try await room.messages.update(
+  newMessage: originalMessage.copy(text: "my updated text"),
+  description: "Message update by user",
+  metadata: nil
+)
+```
+
+```[kotlin]
+val originalMessage: Message
+val updatedMessage = room.messages.update(
+  originalMessage.copy(text = "my updated text"),
+  operationDescription = "Message update by user",
+)
+```
+
 h3(#filter-updates). Filter for updates
 
-blang[javascript].
-  Use the "@messages.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Messages.html#subscribe method to receive messages in a room. To filter for updated messages, provide a listener that checks the @type@ property of the message event:
+blang[javascript,swift,kotlin].
+  Use the <span lang="javascript">"@messages.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Messages.html#subscribe</span><span lang="swift">"@messages.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-swift/main/AblyChat/documentation/ablychat/messages/subscribe()-8jolq</span><span lang="kotlin">"@messages.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-kotlin/main/dokka/chat-android/com.ably.chat/-messages/subscribe.html</span> method to receive messages in a room. To filter for updated messages, provide a listener that checks the <span lang="javascript">@type@</span><span lang="swift">@action@</span><span lang="kotlin">@type@</span> property of the message event:
```

Comment:
There appears to be a bit of odd formatting here around `8jolq` for Swift

Reply:
Fixed